src/app/root.module.ts

```diff
@@ -18,7 +19,8 @@ import { NavigationModule } from './shared/navigation/navigation.module';
     NavigationModule,
     HttpClientModule,
     ApplicationFrameModule,
-    ObservabilityDashboardModule
+    ObservabilityDashboardModule,
+    MatSnackBarModule
```

Comment:
This shouldn't be provided directly, components which use it should import our snackbar module - the usage of material should be isolated inside that mmodule.

Reply:
I was actually trying to do this. But interestingly enough, without doing this import at app level, the snackbar didn't work. I'm still trying to work out why.

Reply:
Hmm - this is likely to do with the a root service getting a root injector, which is not being able to find a lazy loaded component. Instead of providing the service in root, provide it in the module - see how snackbar itself is written: https://github.com/angular/components/blob/master/src/material/snack-bar/snack-bar.ts#L48

Reply:
@aaron-steinfeld Tried that as well. Still doesn't work without importing MatSnackbarModule in the root module. Possible that I'm missing something.

Reply:
Will take a look

Reply:
Thank you!